src/main/resources/feedbackQuestionRubricEditFormBody.html

```diff
@@ -1,12 +1,12 @@
-<tr id="rubricRow-${qnIndex}-${row}">
+<tr id="rubricRow-${qnIdx}-${row}">
```

Comment:
I prefer the longer version. We should use the full word camel case wherever possible.